### PR TITLE
remove-commented-code-fix-quantity

### PR DESCRIPTION
**Pull request by lionllopez**
Removes commented code and fixes quantity issue. Also fixes update order function.

### CODE REVIEW

File: routes/orders.js [hunk 1]
```diff
@@ -73,24 +73,24 @@ ordersRouter.post("/", requireUser, async (req, res, next) => {
     }
 });
 
-ordersRouter.patch("/:orderId", async (req, res, next) => {
+ordersRouter.patch("/:orderId", requireUser, async (req, res, next) => {
```
Comment (frankjoesilva) [inline routes/orders.js:75]: needs requireAdmin

File: routes/orders.js [hunk 2]
```diff
@@ -73,24 +73,24 @@ ordersRouter.post("/", requireUser, async (req, res, next) => {
...
-        const order = await updateOrder({ orderId, status, userId });
+        const order = await updateOrder(orderId, {status});
         res.send(order);
     } catch (error) {
         next(error);
     }
 });
 
-ordersRouter.delete("/:orderId", requireAdmin, async (req, res, next) => {
+ordersRouter.delete("/:orderId", requireUser, async (req, res, next) => {
```
Comment (frankjoesilva) [inline routes/orders.js:87]: need requireAdmin

**Review by frankjoesilva — CHANGES_REQUESTED** [review]

File: routes/orders.js (hunk 1 above)
Comment (lionllopez) [reply routes/orders.js:75]: I thought we agreed we were just going to leave it like this for now so the user could update the order at checkout to 'completed'.

Comment (dennisccastro) [reply routes/orders.js:75]: Has this issue been resolved?

File: routes/orders.js (hunk 2 above)
Comment (dennisccastro) [reply routes/orders.js:87]: And this too?

**Review by dennisccastro — APPROVED** [review]

**Review by frankjoesilva — APPROVED** [review]

**Pull request merged**